### ISSUE TITLE
Google sign out button

### ISSUE THREAD

**Issue by jrobinso** [igvteam/igv-webapp#224]
We need the ability to sign out of Google, or more ideally switch user.   We'll start with simple sign out, more functionality can be added later.

This could be via a pulldown menu, right-justified, perhaps titled with s smallish icon, not text.  In the future we can change this icon to be the user's avatar from their google profile.  On clicking the menu should open with, at present, a single item -- "Sign out".   We can enable / disable this item depending on user's current sign in state   The following should work to determine if the user is signed in.  

```
gapi.auth2.getAuthInstance().currentUser.get().isSignedIn()
```

The GoogleAuth utilities has a method to sign out

```
import {GoogleAuth, igvxhr} from '../node_modules/igv-utils/src/index.js';
GoogleAuth.signOut();
```

We can't really have a "sign in" option here as we don't know the scope the user intends to access  (Google Drive or Google Cloud Storage).

The menu should only be visible if Google is enabled.  This is determined in the main function of app.js as follows

```
    const enableGoogle = (config.clientId  || config.apiKey) &&
        (window.location.protocol === "https:" || window.location.host === "localhost");
```

**Comment by turner** [comment]
@jrobinso is there a callback available that fires after the sign in?

**Comment by jrobinso** [comment]
@helgathorv @turner reopening for cosmetic tweaks

**Comment by turner** [comment]
Fixed with commit 36286828
